redirector.html

```diff
@@ -166,6 +174,9 @@ <h4><span class="disabled-marker" ng-show="r.disabled">[Disabled] </span><span>{
 						<div ng-show="r.excludePattern">
 							<label>excluding:</label><p>{{r.excludePattern}}</p>
 						</div>
+						<div ng-if="r.patternDesc">
+							<label>Hint:</label><p>{{r.patternDesc}}</p>
```

Comment:
(Labelled it "Hint" here as "Pattern Description" seemed lengthy and didn't fit with other labels)